### PR TITLE
Update module-compatibility.md

### PR DESCRIPTION
**Pull request by doctordns**
Added AD Deployment module to the list
Updated Update services to show not supported natively OR in compat layer
Updated Server manager to show it is working with compatibility layer.
Updated Best Practices to not working.

# PR Summary
Added details of working modules from my testing

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7.x preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [X] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [ ] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [ ] PR has a meaningful title
- [ ] PR is targeted at the _staging_ branch
- [ ] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [ ] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/docs-conceptual/whats-new/module-compatibility.md [hunk 1]
```diff
@@ -74,13 +74,14 @@ administrator**.
 | Module name                        | Status                               | Supported OS                       |
 | ---------------------------------- | ------------------------------------ | ---------------------------------- |
 | ActiveDirectory                    | Natively Compatible                  | Windows Server 1809+ with RSAT-AD-PowerShell<br>Windows 10 1809+ with Rsat.ActiveDirectory.DS-LDS.Tools |
+| AD Deployment                      | Works with Compatibility Layer    |                                    |
```
Comment (sdwheeler) [inline reference/docs-conceptual/whats-new/module-compatibility.md]: What OS did you test with?

File: reference/docs-conceptual/whats-new/module-compatibility.md [hunk 2]
```diff
@@ -74,13 +74,14 @@ administrator**.
...
 | ADFS                               | Untested with Compatibility Layer    |                                    |
 | AppBackgroundTask                  | Natively Compatible                  | Windows 10 1903+                   |
 | AppLocker                          | Untested with Compatibility Layer    |                                    |
 | AppvClient                         | Untested with Compatibility Layer    |                                    |
 | Appx                               | Natively Compatible                  | Windows Server 1809+<br>Windows 10 1809+ |
 | AssignedAccess                     | Natively Compatible                  | Windows 10 1809+                   |
-| BestPractices                      | Untested with Compatibility Layer    |                                    |
+| BestPractices                      | Does noe work natively or with Compatibility Layer    |                                    |
```
Comment (sdwheeler) [inline reference/docs-conceptual/whats-new/module-compatibility.md]: What did you OS versions did you test on?
```suggestion
| BestPractices                      | Not working with Compatibility Layer   |                                    |
```

File: reference/docs-conceptual/whats-new/module-compatibility.md [hunk 3]
```diff
@@ -161,7 +162,7 @@ administrator**.
 | ScheduledTasks                     | Natively Compatible                  | Windows Server 1809+<br>Windows 10 1809+      |
 | SecureBoot                         | Natively Compatible                  | Windows Server 1809+<br>Windows 10 1809+      |
 | ServerCore                         | Untested with Compatibility Layer    |                                               |
-| ServerManager                      | Untested with Compatibility Layer    |                                               |
+| ServerManager                      | Works with Compatibility Layer    |                                               |
```
Comment (sdwheeler) [inline reference/docs-conceptual/whats-new/module-compatibility.md]: What OS versions did you test with?

File: reference/docs-conceptual/whats-new/module-compatibility.md [hunk 4]
```diff
@@ -183,7 +184,7 @@ administrator**.
 | TroubleshootingPack                | Natively Compatible                  | Windows 10 1903+                              |
 | TrustedPlatformModule              | Natively Compatible                  | Windows Server 1809+<br>Windows 10 1809+      |
 | UEV                                | Natively Compatible                  | Windows Server ??Future version of Server with Desktop Experience??<br>Windows 10 1903+ |
-| UpdateServices                     | Not working with Compatibility Layer |                                               |
+| UpdateServices                     | Does mpt work natively or with Compatibility Layer |                                               |
```
Comment (sdwheeler) [inline reference/docs-conceptual/whats-new/module-compatibility.md]: Edit not needed. That is what "Not working with Compatibility Layer" means

**Comment by sdwheeler** [comment]
@doctordns The status values are untested, not working, or works natively. If the status is "not working" then that means it does not work natively or with compatibility layer.

**Comment by doctordns** [comment]
> @doctordns The status values are untested, not working, or works natively. If the status is "not working" then that means it does not work natively or with compatibility layer.

So how do you tag modules that just do NOT work in PowerShell 7.

File: reference/docs-conceptual/whats-new/module-compatibility.md (hunk 3 above)
Comment (doctordns) [reply reference/docs-conceptual/whats-new/module-compatibility.md]: 1809

File: reference/docs-conceptual/whats-new/module-compatibility.md (hunk 1 above)
Comment (doctordns) [reply reference/docs-conceptual/whats-new/module-compatibility.md]: 1809

**Comment by doctordns** [comment]
One day I will learn to type more accurately and see my errors. Thanks for the assistance.

**Comment by joeyaiello** [comment]
Thanks for the contributions, @doctordns! Just had a few minor nits

**Comment by doctordns** [comment]
No problems. I feel we should get this right.

File: reference/docs-conceptual/whats-new/module-compatibility.md [hunk 5]
```diff
@@ -74,13 +74,14 @@ administrator**.
 | Module name                        | Status                               | Supported OS                       |
 | ---------------------------------- | ------------------------------------ | ---------------------------------- |
 | ActiveDirectory                    | Natively Compatible                  | Windows Server 1809+ with RSAT-AD-PowerShell<br>Windows 10 1809+ with Rsat.ActiveDirectory.DS-LDS.Tools |
+| AD Deployment                      | Works with Compatibility Layer    |  Windows Server 2019 1809+                                  |
```
[the remark this answers is not in the document]
Comment (doctordns) [reply reference/docs-conceptual/whats-new/module-compatibility.md]: Module name is ADDSDeployment

**Pull request merged**